Cargo.toml

```diff
@@ -27,7 +27,7 @@ thiserror = "1.0"
 http = "0.2"
 itertools = "0.9"
 log = "0.4"
-oauth2 = { version = "4.0.0-alpha.3", default-features = false }
+oauth2 = { version = "4.0.0-alpha.4", default-features = false }
```

Comment:
mind pinning this to `=4.0.0-alpha.4`? it looks like the semver logic considers `4.0.0-alpha.4` to satisfy `4.0.0-alpha.3`, which is causing the current CI failures on the main branch. adding the `=` should prevent that going forward.

Reply:
Done.